src/cpp/include/openvino/genai/image_generation/autoencoder_kl.hpp

```diff
@@ -44,6 +44,30 @@ class OPENVINO_GENAI_EXPORTS AutoencoderKL {
                   const std::string& device,
                   const ov::AnyMap& properties = {});
 
+    AutoencoderKL(const std::string &vae_decoder_model,
+                  const Tensor &vae_decoder_weights,
+                  const Config& vae_decoder_config);
+
+    AutoencoderKL(const std::string &vae_encoder_model,
+                  const Tensor &vae_encoder_weights,
+                  const std::string &vae_decoder_model,
+                  const Tensor &vae_decoder_weights,
+                  const Config& vae_decoder_config);
+
+    AutoencoderKL(const std::string &vae_decoder_model,
+                  const Tensor &vae_decoder_weights,
+                  const Config& vae_decoder_config,
+                  const std::string& device,
+                  const ov::AnyMap& properties = {});
+
+    AutoencoderKL(const std::string &vae_encoder_model,
```

Comment:
when you will port current PR to master, please, add templated version as well like:
```c++
template <typename... Properties,
              typename std::enable_if<ov::util::StringAny<Properties...>::value, bool>::type = true>
    AutoencoderKL(const std::string &vae_decoder_model,
                  const Tensor &vae_decoder_weights,
                  const Config& vae_decoder_config,
                  const std::string& device,
                  Properties&&... properties);
```